adapters/tappx/tappx.go

```diff
@@ -51,14 +51,40 @@ func (a *TappxAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapt
 			Message: "Error parsing bidderExt object",
 		}}
 	}
-
 	var tappxExt openrtb_ext.ExtImpTappx
 	if err := json.Unmarshal(bidderExt.Bidder, &tappxExt); err != nil {
 		return nil, []error{&errortypes.BadInput{
 			Message: "Error parsing tappxExt parameters",
 		}}
 	}
 
+	type Bidder struct {
+		Tappxkey string   `json:"tappxkey"`
+		Mktag    string   `json:"mktag,omitempty"`
+		Bcid     []string `json:"bcid,omitempty"`
+		Bcrid    []string `json:"bcrid,omitempty"`
+	}
+	type Ext struct {
+		Bidder `json:"bidder"`
+	}
+	ext := Ext{
+		Bidder: Bidder{
+			Tappxkey: tappxExt.TappxKey,
```

Comment:
Yes we know. We want in the first place (1) because is the standard place for this parameter in our side and we are going to don't accept the request that send in other place. We still sending the (2) place because if we need to know or analyze what are sending our partners in the bidder of prebid.
If is not important, we prefer now to have the tappxkey in both places.